packages/kauri-wallet/src/app/wallet/wallet.component.html

```diff
@@ -0,0 +1,36 @@
+<div>
+
+  <h1>Wallet Component</h1>
+
+  <select id="currency" name="currency" #currency>
+    <option value="">---Select Currency---</option> <!-- not selected / blank option -->
+    <option value="NAV">NAV</option>
+    <option value="BTC">BTC</option>
+  </select>
+
+  <br>
+
+  <h2>Enter Addresses</h2>
+
+  <br>
+
+  <input matInput placeholder="Enter Address" type="text" #addressOne>
+
+  <br>
+
+  <input matInput placeholder="Enter Address" type="text" #addressTwo>
```

Comment:
We still have a few `matInput`, `mat-raised` etc in the template files

Reply:
fixed